### DISCUSSION TITLE
Use Directory Opus instead of Windows Explorer

### DISCUSSION THREAD

**Discussion by Aholicknight** [winsiderss/systeminformer#1139] [Questions / Answers] [answered]
https://user-images.githubusercontent.com/7843719/155406503-6d661e6b-38e8-4256-9f90-bab233226089.mp4

I am using the nightly build version 3.0.4599 (15436cf), and every time I open a folder, it automatically opens in Windows Explorer. I want it to open in Directory Opus instead; how can I do this? Thank you!

**Comment by dmex** [comment, marked as answer]
>I want it to open in Directory Opus instead

You need to change the `FileBrowseExecutable` setting.

**Reply by Aholicknight** [reply]
Yes, but what would I replace it with?

**Reply by dmex** [reply]
Opus is something like `C:\<Path_To_Opus>.exe /cmd Go "%s"` (you'll need to change `Path_To_Opus` to the actual directory).

**Reply by Aholicknight** [reply]
These works thank you!
